scripts/xtensa-build-zephyr.py

```diff
@@ -325,13 +325,13 @@ def west_init_if_needed():
 	if result_rootdir.returncode != 0:
 		execute_command(["west", "init", "-l", f"{SOF_TOP}"], cwd=west_top)
```

Comment:
Usually we say:

```Fixes: 608833c ("xtensa-build-zephyr.py: use west manifest")```

Reply:
Already merged now, thanks I will remember this in future.